j5/components/buzzer.py

```diff
@@ -0,0 +1,30 @@
+"""Classes for the Buzzer support."""
+
+from abc import ABCMeta, abstractmethod
+
+from j5.boards import Board
+from j5.components import Component
+
+
+class BuzzerInterface(metaclass=ABCMeta):
+    """An interface containing the methods required to control an buzzer."""
+
+    @abstractmethod
+    def buzz(self, board: Board, identifier: int, length: int,
+             frequency: int = None, note: str = None) -> None:
+        """Set the state of an buzzer."""
+        raise NotImplementedError  # pragma: no cover
+
+
+class Buzzer(Component):
```

Comment:
Let's change this to `Piezo`

Reply:
(For the record, the reasoning is that buzzers and piezos are slightly different concepts. A buzzer will produce a continuous tone in response to a continuous voltage being applied, while a piezo requires a PWM signal. Hence the pitch of a piezo can be varied by varying the PWM frequency, while the pitch of a buzzer is fixed.)